configure.js

```diff
@@ -0,0 +1,54 @@
+const { setWeb3 } = require('./src/configure-web3');
+
+let loadedConfig;
+
+function configure (config) {
+  if (!config) {
+    // if there already is a loaded config keep it
+    if (!loadedConfig) {
+      defaultConfigure();
+      loadedConfig = 'default';
+    }
+  } else {
+    if (loadedConfig) {
+      let errorMessage = 'Cannot configure openzeppelin-test-helpers a second time.';
+
+      if (loadedConfig === 'default') {
+        errorMessage += `
+
+A configuration has been loaded by default. Make sure to do custom configuration before importing the library.
```

Comment:
:heart_eyes:

Reply:
:smile: I like the error messages. I don't like the source code in this file so much...